benchmarks/src/main/java/org/corfudb/benchmarks/util/DataUnit.java

```diff
@@ -0,0 +1,37 @@
+package org.corfudb.benchmarks.util;
+
+public enum DataUnit {
```

Comment:
Can't we use something along the lines of storage-units?

Reply:
I don't see the reasons for that.
I have found https://github.com/sebhoss/storage-units on github. It has 4 stars. Also my implementation is just 38 LOC. Also, I'm not sure that the 3rd party library in maven central.  And I really don't the idea to a dependency to an unknown library